Makefile.am

```diff
@@ -8,7 +8,12 @@ if BUILD_OCTAVE_MOD
 octave_dir = Octave
 endif
 
-SUBDIRS = M4 man doc Win32 src $(octave_dir) examples regtest tests programs
+SUBDIRS = M4 Win32 src examples tests
+
+if FULL_SUITE
+SUBDIRS += man doc $(octave_dir) regtest programs
+endif
+
 DIST_SUBDIRS = M4 man doc Win32 src Octave examples regtest tests programs
```

Comment:
I had that initially, but the problem is that when the full suite is disabled, `make check` will fail. Practically, it doesn't matter, as the `check_` prefix prevents anything being built in `examples/`